src/model/GetParticipantIdResponse.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+/**
+ * A response model representing the identifier of a participant
+ *
+ * Example:
+ *
+ * ```
+ * {
+ *    participantId: "some-participan-id"
```

Comment:
```suggestion
 *    participantId: "some-participant-id"
```
My fault.

Reply:
No worries easy to overlook. I am doing two things at once.